integration/deploy/pipeline_test.go

```diff
@@ -154,6 +165,38 @@ func TestDeployPipelineManifestInsidePipeline(t *testing.T) {
 	require.True(t, k8sErrors.IsNotFound(err))
 }
 
+// TestDeployPipelineAndConsumerEnvsFromDependency tests the following scenario:
+// - Deploying a dependency
+// - Consume dependency's variable from main pipeline
+func TestDeployPipelineAndConsumerEnvsFromDependency(t *testing.T) {
```

Comment:
e2e using envs in main in two different ways:
- using envs from dependency `--var` variable
- using envs from dependency `OKTETO_ENV`

Reply:
suggestion -> https://github.com/okteto/okteto/pull/3593#discussion_r1185806556

Reply:
suggestion done 9d45dc7e2c0a589e4648e41982a19cc143347704